packages/core/core/src/PackagerRunner.js

```diff
@@ -351,9 +359,7 @@ export default class PackagerRunner {
         ? url.format(url.parse(sourceRoot + '/'))
         : undefined,
       inlineSources,
-      inlineMap:
```

Comment:
inlineMap has been deprecated in favor of format, as we have 3 formats now and seemed less confusing than having 2 properties start with inline